app/models/flood_risk_engine/enrollment.rb

```diff
@@ -78,6 +78,12 @@ def state_machine
       withdrawn: 6        # BO: used to hide anything created in error
     }
 
+    def submit
+      return if submitted_at.present?
+      self.submitted_at = Time.zone.now
+      pending!
```

Comment:
Should the `pending!` method be called on submission even when there is an existing `submitted_at`?